WordPress/Jetpack/AppConstants.swift

```diff
@@ -4,6 +4,7 @@ struct AppConstants {
     static let productTwitterHandle = "@jetpack"
     static let productTwitterURL = "https://twitter.com/jetpack"
     static let productBlogURL = "https://jetpack.com/blog"
+    static let ticketSubject = "Jetpack for iOS Support"
```

Comment:
Should we use NSLocalizedString like we do for WordPress?

```suggestion
    static let ticketSubject = NSLocalizedString("Jetpack for iOS Support", comment: "Subject of new Zendesk ticket.")
```

Reply:
We'll keep it without localization while we don't change the script that does that for us, it was intentional. :)